rocketpy/rocket/aero_surface.py

```diff
@@ -873,46 +863,54 @@ def evaluate_lift_coefficient(self):
             )
 
             # Differentiating at alpha = 0 to get cl_alpha
-            clalpha2D_incompressible = self.airfoil_cl.differentiate_complex_step(
+            clalpha2D_incompressible = self.airfoil_cl.differentiate_complex_step(  # pylint: disable=invalid-name
                 x=1e-3, dx=1e-3
             )
 
             # Convert to radians if needed
             if self.airfoil[1] == "degrees":
-                clalpha2D_incompressible *= 180 / np.pi
+                clalpha2D_incompressible *= 180 / np.pi  # pylint: disable=invalid-name
 
         # Correcting for compressible flow (apply Prandtl-Glauert correction)
         clalpha2D = Function(lambda mach: clalpha2D_incompressible / self._beta(mach))
 
         # Diederich's Planform Correlation Parameter
-        FD = 2 * np.pi * self.AR / (clalpha2D * np.cos(self.gamma_c))
+        planform_correlation_parameter = (
+            2 * np.pi * self.AR / (clalpha2D * np.cos(self.gamma_c))
+        )  # pylint: disable=invalid-name
```

Comment:
Speaking as someone that opens a PR every week but hasn't review any technical equation in the last few months.
I personally prefer the more readable way. I don't really know what `FD` means.
Making it explicit is better than implicitly.

But I understand this can be quite personal.

Still, I think you will agree with me that the fins' equations are a "hard-to-read" code block.

Reply:
I'm reading this again. 
I honestly can't agree that `self.AR` is more readable and less error-prune than `self.aspect_ratio`

Reply:
A third opinion would be nice